### PR TITLE
stat_summary: inline comments & README.md

### LINKED ISSUE #28: Finish Function 2 - Stat Summary
Please finish function 2: 

- the "statistic summary" of the 4 metrics
- unit test

### PR DESCRIPTION
**Pull request by beth-ouyang**
Main modifications
- Updated `README.md` to include **Developer Installation Instructions**
- Added inline comments to `test_stat_summary.py`

### CODE REVIEW

**Review by WeilinHan8 — APPROVED** [review]
Looks good to me!

**Comment by WeilinHan8** [comment]
@ianm99 could you please review the Readme.md? Thank you so much!

File: environment.yml
Comment (tangyl92) [inline environment.yml:1]: Sorry for the trouble, please follow use my simplified .yml file, I test it

**Review by tangyl92 — APPROVED** [review]
Please change the .yml, others are good

File: src/dscigametrics/stat_summary.py [hunk 1]
```diff
@@ -21,7 +24,7 @@ def stat_summary(data, campaign_id, start_date, end_date):
     Returns
     -------
     summary_table : pd.DataFrame
-        A pandas dataframe contains the mean, median and standard deviation of the four metrics.
+        A pandas dataframe contains the mean, median and standard deviation of the four metrics of each day.
```
Comment (ianm99) [inline src/dscigametrics/stat_summary.py:27]: shouldn't it calculate metrics from the start date to the end date, not "for each day"?

File: environment.yml
Comment (beth-ouyang) [reply environment.yml:1]: No problem!

File: src/dscigametrics/stat_summary.py (hunk 1 above)
Comment (beth-ouyang) [reply src/dscigametrics/stat_summary.py:27]: Oh! It's actually:
> 4 metrics of each day between the start date to the end date -> mean, median and standard deviation of these data

**Review by tangyl92 — APPROVED** [review]

**Comment by beth-ouyang** [comment]
> Please change the .yml, others are good

The .yml file is also edited!

**Pull request merged**